### PR TITLE
Add Mythical Chain

### PR DESCRIPTION
**Pull request by drew-meshreky-mg**

### CODE REVIEW

**Review by milad237 — APPROVED** [review]
0x302F5F5b0A79b2333888886c7651B7EC0b7855F4

**Review by Uzocoiny-source — APPROVED** [review]
_data/chains/eip155-151.json

File: _data/chains/eip155-201804.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://mythicalgames.com/",
+  "shortName": "myth",
+  "chainId": 201804,
+  "networkId": 201804,
+  "icon": "mythical",
+  "explorers": [{
+    "name": "Mythical Chain Explorer",
+    "url": "https://explorer.mythicalgames.com/",
```
Comment (ligi) [inline _data/chains/eip155-201804.json:19]:
```suggestion
    "url": "https://explorer.mythicalgames.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Pull request merged**